### PR TITLE
multicall: review

### PR DESCRIPTION
**Pull request by pkieltyka**

### CODE REVIEW

File: packages/multicall/src/multicall.ts [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
-export type MulticallConf = {
+export type MulticallOptions = {
+  // number of calls to enqueue before calling.
   batchSize: number,
+
+  // number of calls to batch within a time window (in milliseconds). If 0, will disable timeWindow.
   timeWindow: number,
-  contract: string
+
+  // only batch calls for a specific contract.
```
Comment (Agusx1211) [inline packages/multicall/src/multicall.ts:17]: This contract is the address of the SequenceUtils contracts, maybe we could rename the variable

File: packages/multicall/src/multicall.ts [hunk 2]
```diff
@@ -21,38 +26,39 @@ type QueueEntry = {
...
+  timeWindow: 50
+}
+
 export class Multicall {
-  public static DEFAULT_CONF = {
-    batchSize: 50,
-    timeWindow: 50,
-    contract: ""
-  }
+  public static DefaultOptions = () => DefaultMulticallOptions
```
Comment (Agusx1211) [inline packages/multicall/src/multicall.ts:35]: Why using an arrow function?

**Review by Agusx1211 — APPROVED** [review]

File: packages/multicall/src/multicall.ts (hunk 1 above)
Comment (pkieltyka) [reply packages/multicall/src/multicall.ts:17]: ok, good to kno! I'll update

File: packages/multicall/src/multicall.ts (hunk 2 above)
Comment (pkieltyka) [reply packages/multicall/src/multicall.ts:35]: just for style / short-hand

**Pull request merged**